tools/python/boutiques/bosh.py

```diff
@@ -99,20 +99,25 @@ def execute(*params):
         parser.add_argument("-s", "--stream", action="store_true",
                             help="Streams stdout and stderr in real time "
                             "during execution.")
+        parser.add_argument("-z", "--zenodo", action="store_true",
```

Comment:
Ideally we wouldn't like to have to specify '-z' here, just like Docker can run from an image ID or name. Perhaps we could just prefix Zenodo IDs with `zenodo.`, as done in DOIs minted by Zenodo, to make it clear. In this case, `bosh search` should prefix the IDs with `zenodo.` so that the user could just copy-paste them on their command lines. Then if there is a file starting with `zenodo.`, then too bad, we'll ignore it. Files have to end with `.json` anyway, which won't be the case of a Zenodo id.